content/en/api/dashboard_lists/code_snippets/api-dashboard-list-add-items.py

```diff
@@ -10,25 +10,25 @@
 list_id = 4741
 dashboards = [
     {
-        "type": "custom_screenboard",
-        "id": 1414
+        'type': 'custom_screenboard',
+        'id': 'rys-xwq-geh'
     },
     {
-        "type": "custom_timeboard",
-        "id": 5858
+        'type': 'custom_timeboard',
+        'id': 'qts-q2k-yq6'
     },
     {
-        "type": "integration_screenboard",
-        "id": 67
+        'type': 'integration_screenboard',
+        'id': 'jk9-p5y-ii6'
     },
     {
-        "type": "integration_timeboard",
-        "id": 5
+        'type': 'integration_timeboard',
+        'id': 'nf5-aju-tmr'
     },
     {
-        "type": "host_timeboard",
-        "id": 123456789
+        'type': 'host_timeboard',
+        'id': 'jsb-8sp-iz5'
```

Comment:
For `integration_screenboard`, `integration_timeboard` and `host_timeboard` in `v2`, we just are still using the old numerical ID format, but we just cast it to a string. See an example below:

![image](https://user-images.githubusercontent.com/1262407/57637164-b386ec00-7578-11e9-90fe-5575c56e708f.png)